testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -142,6 +146,9 @@ pub fn naka_neon_integration_conf(seed: Option<&[u8]>) -> (Config, StacksAddress
 
     conf.node.miner = true;
     conf.node.wait_time_for_microblocks = 500;
+    conf.node
+        .stacker_dbs
+        .push(boot_code_id(MINERS_NAME, conf.is_mainnet()));
```

Comment:
The constructor for `Config` needs to ensure that the `MINERS_NAME` contract is present in the list of StackerDBs if the node is mining.  It can't be opted out.  Same will go for `.stackers`.

Reply:
kk will add!

Reply:
I am not sure which contructor you are referencing? do you mean the default one or the one which is created from a ConfigFile? I feel this might be better ina. sep PR.